add-on/TA-Demisto/default/app.conf

```diff
@@ -7,11 +7,11 @@ build = 2
 
 [launcher]
 author = Palo Alto Networks
-version = 4.2.0
+version = 4.2.1
 description = This application provides an alert action to create an incident in Cortex XSOAR.
 
 [id]
-version = 4.2.0
+version = 4.2.1
```

Comment:
need to also update the openapi.json file
as can be seen here https://github.com/demisto/splunk-app/pull/47#discussion_r1609880229